index.html

```diff
@@ -1023,150 +1026,66 @@
+* <dfn>de-identified</dfn>
+* [=third party=]
+* [[?Privacy-Concerned]]
 
-* any [=third party=] receiving such [=data=] is barred (eg. based on legal terms)
-  from sharing it further;
-
-* technical measures exist to prevent re-identification or the joining of
-  different data sets involving this [=data=], notably against timing or
-  k-anonymity attacks; and
-
-* there exist contractual terms between the [=first party=] and [=third party=]
-  describing the limited [=purpose=] for which the data is being shared.
-
-Note that [=controlled de-identified data=], on its own, is not sufficient to render [=data processing=]
-[=appropriate=].
-
-### Rights {#data-rights}
+</aside>
 
-[=People=] retain certain rights over data about themselves, and these rights should be facilitated
-by their [=user agent=] and the [=parties=] they're interacting with. While data rights alone are
-not sufficient to satisfy all [=privacy=] principles for the Web, they do support self-determination
-and help improve accountability. Such rights include:
+<aside class="issue">
+  This section has not been edited for narrative.
+</aside>
 
-* The <dfn data-export="">right to access</dfn> [=data=] about oneself.
+<div class="practice">
 
-This right includes being able to discover data about oneself (implying no databases are kept
-secret) and to review
-what information has been collected or inferred.
+<p><span class="practicelab" id="user-agent-data-sharing">
+User agents should only share and present information about the user that
+the user can reasonably anticipate being shared, and which directly relate to the users overt, immediate goals.
+User agents and sites should be conservative when deciding if information is related to a users goals; the
+longer the chain of reasoning used to justify data collection, the less likely for that data
+collection to be ethical and principled.
+</span></p>
 
-* The <dfn data-export="" data-lt="right to erase">right to erase</dfn> [=data=] about oneself.
+</div>
 
-The right to erase applies whether or not terminating use of a service altogether, though what
-[=data=] can be erased may differ between those two cases. On the Web, [=people=] may wish to erase
-data on their device, on a server, or both, and the distinctions may not always be clear.
+Examples:
 
-* The <dfn data-export="" data-lt="right to portability" data-local-lt="portability">right to
-port</dfn> [=data=], including data one has stored with a [=party=], so it can easily be reused or
-transferred elsewhere.
+* Sharing geo-location data with a map website after a user has clicked a "share my location" button has
+  a small number of "hops" between intent and data.
+* Sharing information about the user's selected DNS resolver with a site, so that the site authors can debug
+  possible issues around tail/uncommon resolvers, has a large number of "hops" between user intent (to do
```

Comment:
Maybe

```suggestion
  possible issues around tail/uncommon resolvers, has more "hops" between user intent (to do
```

Users intend to, well, _use_ the websites they're visiting, and buggy websites prevent that.

Reply:
Can you say more about how `a large number of` -> `more` connects to below concern?

> Users intend to, well, use the websites they're visiting, and buggy websites prevent that.

W/o agreeing or disagreeing, im not sure I understand the connection between the two.  Can you say more?